profiles/api/api.py

```diff
@@ -46,7 +46,13 @@ def _list() -> Response:
 
         response = make_response(json.dumps({'total': total, 'items': profile_list},
                                             default=normalize))
+        response.headers['Cache-Control'] = 'max-age=300, public, stale-if-error=86400,' \
```

Comment:
module-level constant could help?
Or perhaps just a module-level function to apply standard headers to `response`?

Reply:
I have added two decorators and pushed a commit to this branch. Take a look and see if they fit what we need.

They are currently in the modules they are used in, these should probably be moved into a decorators module of some sort. There may be additional things you what to add to them, or additional decorators you what to make, if so they can be stacked.